apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -59,13 +60,14 @@ export class SendMessagePush extends SendMessageBase {
 
     const pushChannel: NotificationStepEntity = command.step;
 
+    const stepData: IPushOptions['step'] = {
```

Comment:
I like this separation. You could also call the object `step` already.

Reply:
I didn't want to confuse it with a real `step`